docs/manual/docs/about/comparison-powershell.md

```diff
@@ -461,7 +461,7 @@ Set-PnPHomePage|SharePointPnPPowerShellOnline|
 Set-PnPHubSite|SharePointPnPPowerShellOnline|[spo hubsite set](../cmd/spo/hubsite/hubsite-set.md)
 Set-PnPIndexedProperties|SharePointPnPPowerShellOnline|
 Set-PnPInformationRightsManagement|SharePointPnPPowerShellOnline|
-Set-PnPInPlaceRecordsManagement|SharePointPnPPowerShellOnline|
+Set-PnPInPlaceRecordsManagement|SharePointPnPPowerShellOnline|[spo site inplacerecordsmanagement set](../cmd/spo/site/site-inplacerecordsmanagement-set.md)
```

Comment:
This cmdlet operates on a web rather than on a site, so we're equivalent of `Disable-PnPInPlaceRecordsManagementForSite` and `Enable-PnPInPlaceRecordsManagementForSite` but not of `Set-PnPInPlaceRecordsManagement`

Reply:
Not quite the case Waldek. `Enable-PnPInPlaceRecordsManagementForSite` changes additional settings from their default state and our command is more like `Set-PnPInPlaceRecordsManagement` since it just enables the feature without changing the default settings.

So the difference between the two is that Enable cmdlet does "The in place records management feature will be enabled and the in place record management will be enabled in all locations with record declaration allowed by all contributors and undeclaration by site admins" in addition for some reason.

https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/enable-pnpinplacerecordsmanagementforsite?view=sharepoint-ps

Ours does not. I know all this because I had to look at the code for both cmdlets before start working on that .